cinema/serializers.py

```diff
@@ -68,13 +79,72 @@ class Meta:
             "movie_title",
             "cinema_hall_name",
             "cinema_hall_capacity",
+            "tickets_available",
         )
 
 
+class TicketSerializer(serializers.ModelSerializer):
+    movie_session = MovieSessionListSerializer(read_only=True)
+
+    class Meta:
+        model = Ticket
+        fields = ("id", "row", "seat", "movie_session", "order")
```

Comment:
you don't need order in fields here - it shouldn't be displayed on list view, and it is not expected on create (order will be created separately and passed to tickets)